build/debian-hyperkube-base/Dockerfile

```diff
@@ -14,6 +14,22 @@
 
 FROM BASEIMAGE
 
+# Do we still need the shortened aliases in /?
```

Comment:
nit: this should probably be TODO(somebody):

Reply:
right

Reply:
created #69896 and linked it as a TODO.